packages/compass-indexes/src/modules/index.ts

```diff
@@ -6,49 +6,33 @@ import { RESET_FORM } from './reset-form';
 import isWritable, {
   INITIAL_STATE as WRITABLE_INITIAL_STATE,
 } from './is-writable';
-import isReadonly, {
-  INITIAL_STATE as READONLY_INITIAL_STATE,
-} from './is-readonly';
 import isReadonlyView, {
   INITIAL_STATE as READONLY_VIEW_INITIAL_STATE,
 } from './is-readonly-view';
 import description, {
   INITIAL_STATE as DESCRIPTION_INITIAL_STATE,
 } from './description';
-import indexes, { INITIAL_STATE as INDEXES_INITIAL_STATE } from './indexes';
-import sortOrder, {
-  INITIAL_STATE as SORT_ORDER_INITIAL_STATE,
-} from './sort-order';
-import sortColumn, {
-  INITIAL_STATE as SORT_COLUMN_INITIAL_STATE,
-} from './sort-column';
+import regularIndexes, {
+  INITIAL_STATE as REGULAR_INDEXES_INITIAL_STATE,
+} from './regular-indexes';
 import error, { INITIAL_STATE as ERROR_INITIAL_STATE } from './error';
```

Comment:
Keeping `error` here as its shared with `create-index` and `drop-index` stores. I did not want to do their own error implementation given the time limitations.

Reply:
Those are separate plugins with their own store, so you can safely move / remove this slice in this particular one

Reply:
just cleaned it up in [e9a34b4](https://github.com/mongodb-js/compass/pull/4798/commits/e9a34b489c28324390d41502bff6aefabbb69e12)

Reply:
Not sure why we need to identical files, but as we are not doing any clean up that's fine 👍